keep/api/tasks/process_background_ai_task.py

```diff
@@ -29,6 +29,28 @@ async def process_correlation(ctx, tenant_id:str):
         },
     )
 
+async def process_summary_generation(ctx, tenant_id: str, incident_id:str):
+    logger.info(
+        f"Background summary generation started, {ALGORITHM_VERBOSE_NAME}",
+        extra={"algorithm": ALGORITHM_VERBOSE_NAME, "incident_id": incident_id},
```

Comment:
I thing here ALGORITHM_VERBOSE_NAME should be different from the `process_correlation`